openc3/lib/openc3/packets/packet_item_limits.rb

```diff
@@ -71,7 +71,7 @@ class PacketItemLimits
     def initialize
       @values = nil
       @enabled = false
-      @state = :STALE
+      @state = nil
```

Comment:
According to the comment above :state, "If the limits are disabled the state changes to nil". Thus since the @enabled = false I think @state should be nil. Also without this all the items default to :STALE and don't change.